webapp/src/App.tsx

```diff
@@ -16,8 +33,15 @@ function App() {
         {/* Leave this route, but don't necessarily display it once you have real features. */}
         <Route path="/created_by" element={<CreatedBy />}>
           {/* Using the same element since it's handling both. Normally you'd probably have a separate one */}
-          <Route path=":displayName" element={<CreatedBy />} />
+          <Route path=":displayName" element={<SoundEffectTWO />} />
+          <Route
+            path="/created_by/yellow-power-ranger/:displayName"
+            element={<SoundEffect />}
+          ></Route>
         </Route>
+        {/* <Route path="/yellow-power-ranger" element={<CreatedBy />}>
+          <Route path=":displayName" element={<CreatedBy />} />
+        </Route> */}
```

Comment:
Commented out code, can you remove this?

On rare occassion it's okay to keep commented code around. If we _really_ need to look at old cold though we can look at our git history!